src/renderers/dom/fiber/__tests__/ReactDOMFiber-test.js

```diff
@@ -1004,6 +1004,77 @@ describe('ReactDOMFiber', () => {
         container,
       );
     });
+
+    it('should not warn when rendering into an empty container', () => {
+      spyOn(console, 'error');
+      ReactDOM.render(<div>'foo'</div>, container);
```

Comment:
(did you mean to render quotes into the string? it doesn't much matter but maybe you just wanted `<div>foo</div>` here)